.storybook/custom-loader.js

```diff
@@ -8,13 +7,6 @@ module.exports = () => {
      * @param {import('@babel/core').PartialConfig} cfg - Passed Babel's 'PartialConfig' object.
      */
     config(cfg) {
-      const existingPlugins = /** @type {Required<import('@babel/core').ConfigItem[]>} */ (cfg.options.plugins || []);
-      const pluginsToInclude = existingPlugins.filter(plugin => {
-        const requestedPlugin = /** @type {NonNullable<typeof plugin['file']>} */ (plugin.file).request;
-
-        return !excludePlugins.includes(requestedPlugin);
-      });
-
```

Comment:
The hack for plugin can be removed now, but should be kept for `@griffel/babel-preset`.